net/strongswan/patches/320-change-user-hotplug-handling.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/_updown/_updown
++++ b/src/_updown/_updown
+@@ -18,7 +18,7 @@
+ 
+ # Add your custom ip rules to the /etc/ipsec.user file if you need that functionality.
+ 
+-[ -e /etc/ipsec.user ] && . /etc/ipsec.user "$1"
++[ -e /sbin/hotplug-call ] && /sbin/hotplug-call ipsec "$1"
```

Comment:
No need to check hotplug-call is present as it's installed by the base-files package
Also use exec /sbin/hotplug-call ipsec "$1" (see https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=package/network/services/dnsmasq/files/dhcp-script.sh;h=86032694c156ba08539ca2513c9259dec47c9e46;hb=HEAD#l11)

Reply:
@dedeckeh Using the exec system call will result in NOT executing the default content of the _updown script. Is this really what we want?

Reply:
@sch-m No that's not what we want; see https://github.com/openwrt/packages/pull/7014